src/프렌즈4블록.js

```diff
@@ -0,0 +1,76 @@
+function solution(m, n, board) {
+  const map = [];
+  let answer = 0;
+  
+  for (let i = 0; i < board.length; i++) {
+      map.push(board[i].split(""));
+  }
```

Comment:
```suggestion
  const map = board.map((element) => element.split(""));
```
이렇게 하면 조금더 가독성이 좋을 꺼 같습니다!